crates/iota-indexer/src/processors/network_metrics_processor.rs

```diff
@@ -25,21 +27,29 @@ where
     S: IndexerAnalyticalStore + Clone + Sync + Send + 'static,
 {
     pub fn new(store: S, metrics: IndexerMetrics) -> NetworkMetricsProcessor<S> {
-        let network_processor_metrics_batch_size =
-            std::env::var("NETWORK_PROCESSOR_METRICS_BATCH_SIZE")
+        let min_network_processor_metrics_batch_size =
+            std::env::var("MIN_NETWORK_METRICS_PROCESSOR_BATCH_SIZE")
                 .map(|s| {
                     s.parse::<usize>()
-                        .unwrap_or(NETWORK_METRICS_PROCESSOR_BATCH_SIZE)
+                        .unwrap_or(MIN_NETWORK_METRICS_PROCESSOR_BATCH_SIZE)
                 })
-                .unwrap_or(NETWORK_METRICS_PROCESSOR_BATCH_SIZE);
+                .unwrap_or(MIN_NETWORK_METRICS_PROCESSOR_BATCH_SIZE);
+        let max_network_processor_metrics_batch_size =
+            std::env::var("MAX_NETWORK_METRICS_PROCESSOR_BATCH_SIZE")
+                .map(|s| {
+                    s.parse::<usize>()
+                        .unwrap_or(MAX_NETWORK_METRICS_PROCESSOR_BATCH_SIZE)
+                })
+                .unwrap_or(MAX_NETWORK_METRICS_PROCESSOR_BATCH_SIZE);
         let network_processor_metrics_parallelism =
-            std::env::var("NETWORK_PROCESSOR_METRICS_PARALLELISM")
+            std::env::var("NETWORK_METRICS_PROCESSOR_PARALLELISM")
```

Comment:
Nit: You might want to rename also the `network_processor` variable names to `network_metrics_processor`.